website/docs/reference/snapshot-properties.md

```diff
@@ -41,3 +41,44 @@ snapshots:
 
 ```
 </File>
+
+<VersionBlock firstVersion="1.9">
+
+## Define snapshots in yml
+
+Defining a snapshot in YAML treats snapshots as a distinct resource type, separate from models, allowing for improved organization and consistency within your code.
+
+To add extra logic to your snapshot, you can break it out into an ephemeral model for cleaner development and testing, and then create a snapshot of that model.
+
+<File name='snapshots/<filename>.yml'>
+
+```yml
+
+# snapshots/my_snapshots.yml
+snapshots:
+  - name: orders_snapshot
+    relation: source('jaffle_shop', 'orders')
+    config:
+      schema: snapshots
+      database: analytics
+      unique_key: id
+      strategy: timestamp
+      updated_at: updated_at
+
+```
+
+</File>
+
```

Comment:
might be helpful to add a summary of the config since its new
```suggestion
In this example, the snapshot configuration tracks changes to the orders table over time. The transformation in this snapshot is:  `select * from {{ source('jaffle_shop', 'orders') }}`.
```